zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/builder/fragment/TableBuilder.kt

```diff
@@ -0,0 +1,75 @@
+package org.hexworks.zircon.api.builder.fragment
+
+import org.hexworks.zircon.api.builder.Builder
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.fragment.Table
+import org.hexworks.zircon.api.fragment.builder.FragmentBuilder
+import org.hexworks.zircon.internal.fragment.impl.table.DefaultTable
+import org.hexworks.zircon.internal.fragment.impl.table.TableColumn
+
+class TableBuilder<M : Any>(private val data: List<M>): FragmentBuilder<Table<M>, TableBuilder<M>> {
```

Comment:
Same as above: can we use an `ObservableList`?

Reply:
I already wanted to do this but somehow forgot. The basics are there with 19518d0c but I feel like I want to further improve it ^^
At least I want to change the TableExample to start out with a table not fully filled and a + and - button to add and remove table entries to demonstrate the observable data.